config/thisroot.sh

```diff
@@ -180,7 +184,8 @@ fi
 SHELLNAME=$(getTrueShellExeName)
```

Comment:
I was looking at the commits coming via email and wondered, what wrong with `SHELLNAME=$(basename $SHELL)`? Is there a case where the above won't work?